### ISSUE TITLE
Weird looking losses

### ISSUE THREAD

**Issue by BankNatchapol** [RVC-Boss/GPT-SoVITS#715]
I'm fine-tuning the model for another language using 400 hours of data, with a batch size of 8 and a learning rate of 1e-5. However, the losses after two epochs seem odd. Any recommendations or insights on what might be going wrong?
![image](https://github.com/RVC-Boss/GPT-SoVITS/assets/53829777/ea51f6cc-40aa-4660-8637-af20f5322372)

Thank you so much in advance. 🙏🙏

**Comment by huangxu1991** [comment]
s1_train or s2_train?

**Comment by BankNatchapol** [comment]
> s1_train or s2_train?

It's S2 train. The SoVITS part.
